samples/cinema_video.json

```diff
@@ -0,0 +1,8 @@
+{
+    "_id": "4e1a31e14b032f2fa9e161ee9b009125",
+    "dc_title": "Video Buster Keaton (Demo)",
+    "dc_creator": "Buster Keaton",
+    "dc_issued": "2023-02-21",
+    "text": " [![](https://www.profession-spectacle.com/wp-content/uploads/2016/01/Buster-Keaton.jpg)](https://www.youtube.com/watch?v=JRXkAhMYKEc)",
+    "dc_license": "https://creativecommons.org/licenses/by-sa/4.0/"
```

Comment:
I doubt Buster Keaton chose a license created in 2001... ;)
It is either in the public domain (`CC-0`) either with no license ("All rights reserved" should be displayed).
If you are not sure of the legal status of the movie, choose the latter.

Reply:
It's done